_data/committers.yml

```diff
@@ -47,6 +47,14 @@
   role: PMC
   alias: apitrou
   affiliation: Independent / Ursa Computing
+- name: Ben Kietzman
+  role: PMC
+  alias: bkietz
+  affiliation: Ursa Computing
```

Comment:
A bit a separate issue, but all "Ursa Computing" entries should actually be changed to "Voltron Data"

Reply:
Updated

Reply:
Thanks!